explorer/package.json

```diff
@@ -3,20 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "dev:ncpi-catalog-dug": "./scripts/dev.sh ncpi-catalog-dug && next dev",
```

Comment:
Hi @BruceRodrigues, this is interesting but let's create a separate ticket for this. I have build scripts that rely on these at the moment and I would like to not have to update them for now. Can you revert this change that removes all the scripts? Thanks!

Reply:
Sure thing!